api/v1beta1/azurecluster_validation_test.go

```diff
@@ -899,6 +900,10 @@ func TestValidateAPIServerLB(t *testing.T) {
 						Name: "ip-2",
```

Comment:
Can we add a test case here that tests too many private IPs as well?

Reply:
added via 2ca1da55aaf27f391fcbd220fef28fe3494adcc6